drivers/hmis/app/graphql/types/hmis_schema/enrollment.rb

```diff
@@ -129,6 +129,15 @@ def self.configuration
 
     field :reminders, [HmisSchema::Reminder], null: false
 
+    field :open_enrollment_summary, [HmisSchema::EnrollmentSummary], null: false
+
+    def open_enrollment_summary
+      return [] unless current_user.can_view_open_enrollment_summary_for?(object)
+
+      client = load_ar_association(object, :client)
+      load_ar_association(client, :enrollments).where.not(id: object.id).open_on_date
```

Comment:
Ugh I didn't realize that `open_on_date` will include enrollments that exited on the current date. We don't want that in this case. I guess we really should add another scope.  Sorry for flip flopping!

The below should suffice. Exit dates can't be in the future.
```
scope :open, -> { left_outer_joins(:exit).where(ex_t[:ExitDate].eq(nil)) }
```

It looks like we have a scope called "active" with the below definition, which makes me a little nervous. I can see somone using it to try to get all the open enrollments, not realizing it filters out WIP enrollments. Could you rename it to something clearer like `open_not_in_progress` or `open_non_wip` or `open_excluding_wip`?  I think you added it for usage in `drivers/hmis/app/models/hmis/filter/enrollment_filter.rb` but I'm not sure if its used anywhere else at this point.
```
scope :active, -> { left_outer_joins(:exit).where(ex_t[:ExitDate].eq(nil)).not_in_progress }
```

Reply:
I think we probably want to avoid `open` as a scope, pretty sure that's going to be a reserved word.